web/templates/base.html

```diff
@@ -70,23 +81,35 @@
 
 		<footer>
 			<div class="container">
-					<div class="col-md-4">
+					<div class="col-md-2">
+						<h4 class="h4-footer">SUPPORT</h4>
 						<ul class="footer-links">
-							<li><a href="{% url 'pages-about' %}">Sponsors</a></li>
+							<li><a href="{% url 'pages-sponsors' %}">Sponsors</a></li>
 							<li><a href="{% url 'pages-about' %}">Privacy</a></li>
-							<li><a href="{% url 'pages-about' %}">Support</a></li>
+							<li><a href="mailto:someone@example.com?Subject=TrackCat-Webmaster%20e-mail" target="_top">Webmaster</a>
 						</ul>
 					</div> <!-- end col-md-5 -->
-					<div class="col-md-4">
-						<a href="{% url 'pages-index' %}"><img src="{% static "img/CodeCatz.jpg" %}"/></a>
+					<div class="col-md-3">
+						<h4 class="h4-footer">FOLLOW CodeCatz</h4>
+						<a href="https://www.facebook.com/groups/codecats/"><i class="fa fa-facebook-square fa-3x"></i></a>
+						<a href="https://twitter.com/codecatz"><i class="fa fa-twitter-square fa-3x"></i></a>
+						<a href="https://github.com/CodeCatz"><i class="fa fa-github-square fa-3x"></i></a>
 					</div> <!-- end col-md-2 -->
 					<div class="col-md-4">
-  						<p><strong>CodeCatz</strong><br>
-  							RAMPA Laboratorij<br>
+  						<h4 class="h4-footer">WHERE WE MEET</h4>
+  						<p style="font-size: 18px">	RAMPA Laboratorij<br>
```

Comment:
I suggest instead of inline style, to use a class or an ID to style your paragraph.